### ISSUE TITLE
Not build with new flutter 3.22.0

### ISSUE THREAD

**Issue by marcellocamara** [SachinGanesh/screenshot#181]
```
Error (Xcode): ../../.pub-cache/hosted/pub.dev/screenshot-2.3.0/lib/screenshot.dart:164:9: Error: No named parameter with the name 'size'.
```

The `size` param is removed from the new version of Flutter `3.22.0`

Flutter 3.19.6:
![Captura de Tela 2024-05-13 às 23 01 47](https://github.com/SachinGanesh/screenshot/assets/18273071/ef4d97b6-8b88-469d-a08b-2fe15ba0fbee)

Flutter 3.22.0:
![Captura de Tela 2024-05-13 às 23 02 07](https://github.com/SachinGanesh/screenshot/assets/18273071/94355a1e-b234-428d-b3cd-9efeee887df8)

**Comment by chandrabezzo** [comment]
I see @ritheshSalyan make solve this issue on this [commit](https://github.com/SachinGanesh/screenshot/commit/463c6babc9df2e53ef616279e89408a8bc62b4a8), please publish for new update.

**Comment by marcellocamara** [comment]
> @chandrabezzo: this [commit](https://github.com/SachinGanesh/screenshot/commit/463c6babc9df2e53ef616279e89408a8bc62b4a8), please publish for new update.

Since this commit solves the problem, this approach works until the owner pushes the new version:
```
  screenshot:
    git: https://github.com/SachinGanesh/screenshot.git
```

This uses the master branch with the referenced commit

**Comment by ritheshSalyan** [comment]
Hi everyone , We have published latest version ```screenshot: ^2.4.0``` to pub.dev.  Please use the version from pub.dev. If this solves the issue, Please consider closing the issue.
